README.md

```diff
@@ -27,7 +27,45 @@ Fast and lightweight key-value database library.
 </a>
 </div>
 
-## Features
+---
+
+## 📖 Contents
+- [Features](#-features)
+- [Installation](#-installation)
+- [Usage](#usage)
+- [Synchronous API](#synchronous-api)
+- [API Reference](#api-reference)
+  - [Constructor](#constructor)
+  - [Get](#get)
+  - [Set](#set)
+  - [Exists](#exists)
+  - [Delete](#delete)
+  - [Rename](#rename)
+  - [Get Expire](#get-expire)
+  - [Set Expire](#set-expire)
+  - [Keys](#keys)
+  - [Pagination](#pagination)
+  - [Count](#count)
+  - [Count Expired](#count-expired)
+  - [Multi Get](#multi-get)
+  - [Multi Set](#multi-set)
+  - [Multi Delete](#multi-delete)
+  - [Cleanup](#cleanup)
+  - [Vacuum](#vacuum)
+  - [Flush](#flush)
+  - [Execute](#execute)
+  - [Backup](#backup)
+  - [Restore](#restore)
+  - [Close](#close)
```

Comment:
Thanks for your clarification
It looks better and easy to navigate 

But the table of contents is big 
I don't think we should list each method we can stick to point to the API reference it self

Reply:
Done